source/jormungandr/jormungandr/street_network/utils.py

```diff
@@ -59,3 +59,24 @@ def make_speed_switcher(req):
         FallbackModes.ridesharing.name: req['car_no_park_speed'],
         FallbackModes.taxi.name: req['taxi_speed'],
     }
+
+
+PARK_RIDE_VALUES = {'yes'}
+
+
+def pick_up_park_ride_car_park(pt_objects):
```

Comment:
same here https://github.com/CanalTP/navitia/pull/3228/files#diff-a16a0bc9cd0342e4e2b429a601790fe0R132
the function already exists